File: python/src/main/python/QuesoManchego.py
class QuesoManchego:
    def __init__(self, melted=False, temperature=0, melting=20):
        self.melted = melted
        self.temperature = temperature
        self.melting = melting

    def isMelted(self):
        return self.melted

    def getCurrentTemperature(self):
        return self.temperature

    def getMeltingTemperature(self):
        return self.melting

    def setCurrentTemperature(self, temp):
        self.temperature = temp

    def melt(self, melted):
        self.melted = melted

File: csharp/unittest-practice/src/main/QuesoManchego.cs
namespace unittestpractice.main
{
    internal class QuesoManchego : IQueso
    {
        public bool Melted;
        public int Temperature;
        public int Melting = 10;

        public bool IsMelted()
        {
            return Melted;
        }

        public int GetCurrentTemperature()
        {
            return Temperature;
        }

        public int GetMeltingTemperature()
        {
            return Melting;
        }

        public void SetCurrentTemperature(int temp)
        {
            Temperature = temp;
        }

        public void Melt(bool melted)
        {
            Melted = melted;
        }
    }
}

